lib/dialects/aix/dnode.c

```diff
@@ -155,17 +155,17 @@ struct gnode *ga; /* local gnode address */
         switch (f.set.l_type & (F_RDLCK | F_WRLCK)) {
 
         case F_RDLCK:
-            return ((l) ? 'R' : 'r');
+            return ((l) ? LSOF_LOCK_READ_FULL : LSOF_LOCK_READ_PARTIAL);
```

Comment:
Style question: Any particular reason for those extra parens in the lines below?
```suggestion
            return (l) ? LSOF_LOCK_READ_FULL : LSOF_LOCK_READ_PARTIAL;
```